dev-support/Jenkinsfile_GitHub

```diff
@@ -46,6 +46,7 @@ pipeline {
         TESTS_FILTER = 'cc,checkstyle,javac,javadoc,pylint,shellcheck,whitespace,perlcritic,ruby-lint,rubocop,mvnsite'
         EXCLUDE_TESTS_URL = "${JENKINS_URL}/job/HBase-Find-Flaky-Tests/job/${CHANGE_TARGET}/lastSuccessfulBuild/artifact/excludes"
 
+        ////
```

Comment:
?

Reply:
Trying to distinguish between comments that describe the following section of code vs. comments that describe the following line. Suggestions?